packages/utilities/gel-foundations/src/scripts/arabic.js

```diff
@@ -0,0 +1,88 @@
+const arabicTypographyConfig = {
```

Comment:
These files currently dont have a nice import path, You would have to do 
```
import arabicTypography from '@bbc/gel-foundations/dist/scripts/arabic'
```

Reply:
Hey @DenisBBC im not sure https://github.com/bbc/psammead/pull/353/commits/c75ff29436c8fe161606e881b4ad35b44ca1b7a9 changes anything You still have to reference in the dist folder `@bbc/gel-foundations/dist/scripts/arabic`

Reply:
An example of treeshaking we want is https://survivejs.com/webpack/optimizing/tree-shaking/#demonstrating-tree-shaking

I suggest putting a file at the base of this package called `scripts.js` and inside have

```
module.exports = {
    enGb: require('./dist/scripts/enGB'),
    arabic: require('./dist/scripts/arabic'),
    ...etc
}
```

That allows the user to do 

`import { arabic } from '@bbc/gel-foundations/scripts'`

And tree shake all other typographies

Reply:
@dr3 I added Tree shaking as suggested but when importing the script doing `import { arabic } from '@bbc/gel-foundations/scripts` it returns the script object with a default key, so I had to add a `.default` at the end of each require.